### PR TITLE
MAT-6839 added empty codes on cql editor screen

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-6839](https://jira.cms.gov/browse/MAT-6839)
(Optional) Related Tickets:

### Summary
added empty codes on cql editor screen

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/common/CqlEditorTerminologySection.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+import "./CqlEditorTerminologySection.scss";
+
+interface CqlEditorTerminologySectionProps {
+  title: string;
+  children?: any;
+}
+
+const CqlEditorTerminologySection = (
```
Comment (adongare) [inline src/common/CqlEditorTerminologySection.tsx]: can we rename this to `TerminologySection`.  while importing it is `TerminologySection`. also test and css file name

File: src/common/CqlEditorTerminologySection.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      className="cql-editor-tab-heading"
+      data-testid={`cql-editor-${props.title}-sub-heading`}
+    >
+      <div
+        onClick={() => {
+          setOpen(!open);
+        }}
+        tabIndex={0}
+        role="button"
+        onKeyPress={(e) => {
```
Comment (adongare) [inline src/common/CqlEditorTerminologySection.tsx]: onclick does not cover this?

File: src/common/CqlEditorTerminologySection.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  props: CqlEditorTerminologySectionProps
+) => {
+  const { title, children } = props;
+  const [open, setOpen] = useState(true);
+  const chevronClass = open ? "chevron-display open" : "chevron-display";
+  const growingDivClass = open ? "growing-div open" : "growing-div";
+
+  return (
+    <div
+      className="cql-editor-tab-heading"
```
Comment (adongare) [inline src/common/CqlEditorTerminologySection.tsx]: these IDs/classes I think should be terminology specific and not cql editor

File: src/cqlEditorWithTerminology/CqlEditorWithTerminology.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                inboundErrorMarkers={inboundErrorMarkers}
+                readOnly={readOnly}
+                validationsEnabled={validationsEnabled}
+                setOutboundAnnotations={setOutboundAnnotations}
+              />
+            </div>
+          </div>
+        </Allotment.Pane>
+        <Allotment.Pane>
+          <RightPanel />
```
Comment (adongare) [inline src/cqlEditorWithTerminology/CqlEditorWithTerminology.tsx]: can we name this bit more meaningful.

File: src/common/CqlEditorTerminologySection.tsx (hunk 1 above)
Comment (sb-prateekkeerthi) [reply src/common/CqlEditorTerminologySection.tsx]: Sure, I'll change it

File: src/rightPanel/RightPanel.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import KeyboardTabIcon from "@mui/icons-material/KeyboardTab";
+import { IconButton } from "@mui/material";
+import ValueSetsSection from "./valueSetsTab/ValueSetsSection";
+import RightPanelNavTabs from "./RightPanelNavTabs";
+import DefinitionsSection from "./definitionsTab/DefinitionsSection";
+import CodesSection from "./codesTab/CodesSection";
```
Comment (adongare) [inline src/rightPanel/RightPanel.tsx]: in other projects I think we followed convention- same folder name as conponent name. e.g.`"./codesSection/CodesSection";`

File: src/common/CqlEditorTerminologySection.tsx (hunk 2 above)
Comment (sb-prateekkeerthi) [reply src/common/CqlEditorTerminologySection.tsx]: this is something for 508 compliance, onClick is not working when we click enter on the tab heading so onKeyPress is used

File: src/common/CqlEditorTerminologySection.tsx (hunk 3 above)
Comment (sb-prateekkeerthi) [reply src/common/CqlEditorTerminologySection.tsx]: Sure, I changed them to terminology-section-tab-heading, also IDs to have terminology-section instead of cql-editor

File: src/cqlEditorWithTerminology/CqlEditorWithTerminology.tsx (hunk 4 above)
Comment (sb-prateekkeerthi) [reply src/cqlEditorWithTerminology/CqlEditorWithTerminology.tsx]: Sure, I have changed the name to TerminologySectionPanel

File: src/rightPanel/RightPanel.tsx (hunk 5 above)
Comment (sb-prateekkeerthi) [reply src/rightPanel/RightPanel.tsx]: thank you, i have updated it

**Review by adongare — APPROVED** [review]
